sql/permissions.sql

```diff
@@ -0,0 +1,17 @@
+DROP TABLE IF EXISTS permissions;
+DROP TABLE IF EXISTS position_permissions;
```

Comment:
These `DROP`s need to be switched; you break a foreign key constraint if you drop `permissions` while `position_permissions` still exists